client/src/components/Projects/ProjectsPage.js

```diff
@@ -733,16 +746,29 @@ const ProjectsPage = ({ contentContainerRef }) => {
                     <tr className={classes.tr}>
                       {headerData.map(header => {
                         return (
-                          <td key={header.id}>
+                          <td
```

Comment:
I guess what you are trying to do here is change the header appearance while it's corresponding popup is open.  This is outside the scope of this issue, and will be addressed by a separate issue, but if we wanted to do this, you don't need any new props at all. The reactjs-popup has a feature that allows you to change the trigger (i.e. Column Header appearance, depending on whether the popup is open or not - See the Trigger section on the page  https://react-popup.elazizi.com/component-api